src/datatable/index.jsx

```diff
@@ -9,9 +9,10 @@ import { getValue } from '../utils';
 import DatatableHeader from './header';
 import { Pagination } from '../';
 
-export function Row({ row, schema, Expandable, Actions, expands }) {
+export function Row({ row, schema, Expandable, Actions, expands, alwaysExpanded }) {
   const [expanded, setExpanded] = useState(false);
```

Comment:
Some of the inline logic can be simplified by using the `alwaysExpanded` prop to set the initial state.

With a little rearranging of the lines (to move this below `rowExpands` definition) this can replace the changes at L28/43:

```suggestion
  const [expanded, setExpanded] = useState(rowExpands && alwaysExpanded);
```